App.js

```diff
@@ -25,183 +22,52 @@ export default class App extends Component {
atitude":"50.834811"},"1224689":{"PAON":"25","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":206500,"Longitude":"-0.127671","Latitude":"50.834811"},"1224690":{"PAON":"27","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":189950,"Longitude":"-0.127671","Latitude":"50.834811"},"1224691":{"PAON":"28","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":430000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224692":{"PAON":"29","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":310000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224693":{"PAON":"30","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":390000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224694":{"PAON":"31","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":360000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224695":{"PAON":"32","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":330000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224696":{"PAON":"33","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":358500,"Longitude":"-0.127671","Latitude":"50.834811"},"1224697":{"PAON":"34","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":69000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224698":{"PAON":"35","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":385000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224699":{"PAON":"36","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":390000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224700":{"PAON":"37","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":139950,"Longitude":"-0.127671","Latitude":"50.834811"},"1224701":{"PAON":"39","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":170000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224702":{"PAON":"4","SAON":"BASEMENT FLAT","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":147000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224703":{"PAON":"4","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":160000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224704":{"PAON":"5B","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":280000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224705":{"PAON":"6","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":500000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224706":{"PAON":"7","SAON":null,"Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":502000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224707":{"PAON":"8","SAON":"FLAT 1","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":180000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224708":{"PAON":"8","SAON":"FLAT 3","Street":"CALEDONIAN ROAD","Postcode":"BN2 3HX","Price":245000,"Longitude":"-0.127671","Latitude":"50.834811"},"1224709":{"PAON":"8","SAON":"FLAT 2","Street"
```

Comment:
Do we need a console write? Perhaps something to notify the user, as the app requires it for core functionality

Reply:
These lines of code have been removed. That's why they have been highlighted in red.